### PR TITLE
ECS Terraform example

### PR DESCRIPTION
**Pull request by Marton6**

### CODE REVIEW

**Comment by Marton6** [comment]
@florisvdg can you please create a GitHub banner for this guide as well?

**Comment by florisvdg** [comment]
https://secrethub.io/img/integrations/aws-ec2/github-banner.png

**Comment by Marton6** [comment]
Tested the example.

File: aws/ecs/example.tf [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      ]
+    },
+  ]
+}
+
+variable "vpc_id" {}
+
+variable "subnets" {
+  type = list(string)
+}
```
Comment (florisvdg) [inline aws/ecs/example.tf:107]: See https://github.com/secrethub/examples/pull/30#discussion_r442078757

File: aws/ecs/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+secrethub demo init
+```
+
+To launch an example ECS cluster, run:
+```
+terraform init
+terraform apply
+```
+
+Get the public IP of the cluster from the AWS console or using the AWS CLI and visit `http://<PUBLIC-IP>:8080` to see the app running.
```
Comment (jpcoenen) [inline aws/ecs/README.md:32]: Can't we just give this as a Terraform output?

Comment (Marton6) [reply aws/ecs/README.md:32]: There is no easy way to output this from Terraform, unfortunately.

**Comment by jpcoenen** [comment]
Task linked: [CU-40pcce Add AWS ECS + Terraform to examples repo](https://app.clickup.com/t/40pcce)

File: aws/ecs/README.md (hunk 2 above)
Comment (jpcoenen) [reply aws/ecs/README.md:32]: Ahw :cry:

**Pull request merged**